XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1472,6 +1472,29 @@
       "ImagePath": "",
       "Description": "",
       "Items": [
+        {
+          "UniqueId": "AnimatedVisualPlayer",
+          "Title": "AnimatedVisualPlayer",
+          "Subtitle": "The AnimatedVisualPlayer allows playback of animated Windows.UI.Composition.Visual trees.",
```

Comment:
Maybe this should be "You can use AnimatedVisualPlayer to playback animations" or something. The way that it is currently worded is overly technical and doesn't describe the user value.

Reply:
except it can't playback all animations, only those created with Windows.UI.Composition.

Reply:
True, but the meta-feedback was that this text was overly technical. For example, the Implicit Transitions page says "Use implicit transitions to automatically animate changes to properties" (even though it only supports a few of the UIElement properties) and the ListView page says "The ListView lets you show a collection of items in a list that scrolls vertically". 

What about "AnimatedVisualPlayer allows you to playback Lottie animations" or something?

Reply:
Makes sense. I'll change this to be something without tech specifics.

Reply:
AnimatedVisualPlayer allows you to playback animations such as Lottie?

Reply:
Thanks for the input @improbabilityphysics 😁 - @sohchatt updated the text to mention "motion graphics" specifically so I think we're good now.